app/controllers/hotels_controller.rb

```diff
@@ -0,0 +1,6 @@
+class HotelsController < ApplicationController
+  def show
+    @hotel = Hotel.find(params[:hotel_id])
+    @guests = @hotel.list_of_guests
```

Comment:
Similar to your GuestsController, you can also call the `list_of_guests` method from the view - this is fine though.